openrarity/cli/utils.py

```diff
@@ -6,5 +6,15 @@
 
 
 def print_rankings(ranks: list[RankedToken], columns: list[str]):
+    """
+    Prints the given rank data in the form of a table to STDOUT.
+
+    Parameters
+    ----------
+    ranks : list[RankedToken]
+        Ranks which need to print on STDOUT.
+    columns : list[str]
+        list of the column names.
```

Comment:
Capitalization. And be more descriptive. And add in what column names are available and an example of the input.
> A subset of columns to print for the ranks.

Reply:
fixed it.